static/js/components/EmailCompositionDialog.js

```diff
@@ -11,15 +11,15 @@ import type {
 const createDialogActions = (close, send) => ([
   <Button
     type="button"
-    className="dialog-button cancel-button"
+    className="mdl-button mdl-js-button cancel-button"
```

Comment:
(any button in a JS file where we have `import Button from 'react-mdl/lib/Button';` at the top)